README.md

```diff
@@ -1,66 +1,55 @@
- - nginx ingress controller [link](https://github.com/kubernetes/ingress-nginx/)
- - istio [link](https://istio.io/latest/docs/)
- - monitoring stack [link](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
- - docker and [k3d](https://k3d.io/) to create local k8s cluster
-
-### Architecture 
-
-![Continuous Load](./docs/Continuous%20Load%20Diagram.png)
-
-The load-injector send load to source-podinfo and not to the ingress directly because load-injectors like k6 or gatling are design to retry on connection reset which can hide issues. It is also interresting to own a golang application to go further and add functional validation like JWT secret rotation, MTLS or talking to a database. 
+ - Podinfo: [golang application](https://github.com/stefanprodan/podinfo)
+ - Prometheus Operator [link](https://prometheus-operator.dev/)
+ - Grafana Operator [link](https://grafana-operator.github.io/grafana-operator/)
+ - Helm
 
 ### Monitoring
 
-Each component creates a ServiceMonitor or PodMonitor resource to configure prometheus automatically.
+A ServiceMonitor or PodMonitor resources are used to configure prometheus and are found in the relevant charts.
 
 ### Alerts
 
-Each component creates PrometheusRule resources to configure alerts on prometheus automatically.
-Those alerts follow the [Golden Signal principles](https://sysdig.com/blog/golden-signals-kubernetes/). 
+PrometheusRule resources are created to configure alerts on prometheus.
+Such alerts follow the [Golden Signal](https://sysdig.com/blog/golden-signals-kubernetes/) principles. 
 
 ### Dashboard 
 
-Continuous dashboard is create automatically and the link will be available within the console after running the script below.
+A GrafanaDashboard resource is used to create the continuous load dashboard.
 
-### Run 
+## Run 
 
+To install both the prerequisites and the main chart run:
 ```
-./create-cluster-istio-k3d.sh
+./deploy.sh -i
 ```
+Note:
+If there is already a deployment of the prometheus operator installed on the cluster, running this can lead to a conflict and prevent the prometheus instance from starting.  To fix this, delete one of the operator deployments e.g.
+```
+# find out if there are multiple operator deployments
+kubectl get all --all-namespaces | grep 'prom.*operator'
 
-## Integration tests 
-
-Integration tests have been created [here](podinfo/test/integration/integration_test.go) but will not work for the local version, it needs a bit of work.
+# delete the unnecessary deployment
+kubectl -n <namespace> delete deployment.apps/prometheus-operator
+```
 
+To install only the continuous-load components run without the flag:
 ```
-cd podinfo ; make integration prometheus_url=$prometheus_url source_url=$source_url target_url=$target_url region=$region cluster=$cluster
+./deploy.sh
```

Comment:
add a section on what i do next to see the dashboard, assume this is the first time someone has tried to use this